packages/fontconfig.rb

```diff
@@ -77,6 +77,8 @@ def self.postinstall
       puts 'To complete the installation, execute the following:'.orange
       puts 'source ~/.bashrc'.orange
     end
-    system "env FONTCONFIG_PATH=#{CREW_PREFIX}/etc/fonts fc-cache -fv"
+    # The following postinstall fails if graphite isn't installed when fontconfig
+    # is being installed.
+    system "env FONTCONFIG_PATH=#{CREW_PREFIX}/etc/fonts fc-cache -fv || true"
```

Comment:
Does this mean graphite should be a fontconfig dependency?

Reply:
Wait...graphite is already a dependency?

Reply:
During a build of a package which requires fontconfig, a bunch of depends are getting installed, but there's currently no guarantee that fontconfig's deps will get installed BEFORE fontconfig... 

In this case, fontconfig install started before graphite. postinstall fails because no graphite... the entire build fails... never getting to graphite.

This gets around that. Dependency postinstall errors are currently fatal during builds.

Reply:
Probably a better fix is to make sure all the dependencies are installed before the build.

Reply:
A better fix is needed. This is needed to get builds working properly. 😐